### PR TITLE
6.0.0-beta.7 Change Log

### PR DESCRIPTION
**Pull request by mirichmo**
Initial draft of the Beta.7 change log. Please review and leave comments.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,80 @@
...
+### Engine updates and fixes
+
+* Use stricter rules when unwrapping a PSObject that wraps a COM object (#4614)
+* Remove appended Windows PowerShell `PSModulePath` on Windows. (#4656)
+* Ensure `GetNetworkCredential()` returns null if PSCredential has null or empty user name (#4697)
+* Push locals of automatic variables to 'DottedScopes' when dotting script cmdlets (#4709)
+* Fix `using module` when module has non-terminating errors handled with `SilentlyContinue` (#4711) (Thanks @iSazonov)
+* Enable use of 'Singleline,Multiline' option in split operator (#4721) (Thanks @iSazonov)
+* Fix error message in `ValidateSetAttribute.ValidateElement()` (#4722) (Thanks @iSazonov)
+* Rename $IsOSX to $IsMacOS (#4757)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: We should call this out as a Breaking Change (from previous PSCore6 release)

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mirichmo) [reply CHANGELOG.md]: OK. Should we add the tag to the PR as well?

Comment (SteveL-MSFT) [reply CHANGELOG.md]: Yes, should have been tagged.  I think the issue was tagged, but not the PR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,80 @@
...
+
+### General cmdlet updates and fixes
+
+* Add Meta, Charset, and Transitional parameters to `ConvertTo-HTML` (#4184) (Thanks @ergo3114)
+* Prevent `Test-ModuleManifest` from loading unnecessary modules (#4541)
+* Remove AlternateStream code and `-Stream` from provider cmdlets on non-Windows (#4567)
+* Add explicit ContentType detection to `Invoke-RestMethod` (#4692)
+* Fix for error on `Enter-PSSession` exit (#4693)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: Like other entries, `Fix` here should be a verb, right? If so, it should be changed to `Fix an error`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,80 @@
...
+* Silence PSSessionConfiguration test warning messages in the log (#4794)
+* Add tests for
+    * `Get-Service` (#4773)
+    * `Set-Service` and `New-Service` (#4785)
+    * `Trace-Command` (#4288)
+    * `StaticParameter` (#4779)
+    * `Test-Wsman` (#4771)
+    * `New-Object -ComObject` (#4776)
+    * ProxyCommand APIs (#4791)
+* Disable tests
```
Comment (daxian-dbw) [inline CHANGELOG.md]: > Disable tests

Should it be **Disabled**?

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

**Comment by daxian-dbw** [comment]
Please fix the build failure as well.

File: CHANGELOG.md (hunk 2 above)
Comment (mirichmo) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 3 above)
Comment (mirichmo) [reply CHANGELOG.md]: Disable is a verb, Disabled is an adjective

**Review by SteveL-MSFT — APPROVED** [review]

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**